ajax/libs/email-autocomplete/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "email-autocomplete",
+  "filename": "jquery.email-autocomplete.min.js",
+  "description": "jQuery plugin that displays in-place autocomplete suggestions for email input fields.",
+  "homepage": "https://10w042.github.io/email-autocomplete/demo/",
```

Comment:
Please remove the "homepage" section as it's not really a homepage but just a demo page.